usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -44,6 +46,8 @@
     "place_of_perform_state_code": {"delta": "STRING", "postgres": "TEXT"},
     "place_of_perform_state_name": {"delta": "STRING", "postgres": "TEXT"},
     "place_of_performance_zip": {"delta": "STRING", "postgres": "TEXT"},
+    "place_of_perform_county_code": {"delta": "STRING", "postgres": "TEXT"},
```

Comment:
Why did you move these key value pairs?

Reply:
Those columns were originally derived on the USASpending code and the order of the columns had those last. Now that they are pulled from the Broker, I moved them to the section where we're pulling other broker values directly (sans major transformations/derivations).

Reply:
I'm assuming you are referring to the sub_place_* and the sub_legal_*. The prime award county fields are new.

Reply:
Correct. Why does order matter?

Reply:
It groups the columns into their respective sections for clarity/simplicity (see the comments)

-- Broker Subaward Table Meta
-- Prime Award Fields (from Broker)
-- Subaward Fields (from Broker)
-- Additional Prime Award Fields (from Broker)
-- Additional Subaward Fields (from Broker)
-- USAS Links (and associated derivations)
-- USAS Derived Fields

So we're moving those updated sub county cols from USAS Derived Fields to Subaward Fields. The order of the models at the top must match the order of the columns selected in the populate query.